### PR TITLE
[plugin.video.srf_ch_replay] 2.0.1

### PR DESCRIPTION
**Pull request by ManBehindMooN**
### Checklist:

- [y] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [y] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [y] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.srf_ch_replay/addon.py [hunk 1]
```diff
@@ -87,12 +80,30 @@ def listTvShows():
...
             picture = ''
-        _addShow(title, show['id'], mode, desc, picture, page)
+
+        firstTitleLetter = title[:1]
+        if (firstTitleLetter.lower() == letter) or (not firstTitleLetter.isalpha() and not str(letter).isalpha()):
+            _add_show(title, show['id'], mode, desc, picture, page)
 
     xbmcplugin.addSortMethod(pluginhandle, 1)
     xbmcplugin.endOfDirectory(pluginhandle)
+    if myView:
```
Comment (enen92) [inline plugin.video.srf_ch_replay/addon.py]: Not allowed sorry, plugins can't force skin viewmodes. Please drop this call.

File: plugin.video.srf_ch_replay/addon.py [hunk 2]
```diff
@@ -175,6 +186,9 @@ def listEpisodes(showid, showbackground, page):
     if page < maxpage or maxpage == 0 and len(show) == int(numberOfEpisodesPerPage):
         page = page + 1
         _addnextpage(tr(30005).format(page, maxpage), showid, 'listEpisodes', '', showbackground, page)
+        
+    if myView:
```
Comment (enen92) [inline plugin.video.srf_ch_replay/addon.py]: Same

**Review by enen92 — CHANGES_REQUESTED** [review]
Sorry, the add-on is using functionalities that we don't allow

File: plugin.video.srf_ch_replay/addon.py (hunk 1 above)
Comment (ManBehindMooN) [reply plugin.video.srf_ch_replay/addon.py]: @enen92: Is there an allowed way for that? What I wanna basically do is to have the InfoWall ViewType when I'm in the add-on. Right now the default seems to be the list and for every new view or change a page I have to set the view type again. If there is a way could you please refer me to the documentation? Right now this was the only way I have found... Thank you for your help... if there is no way... I will remove it...

Comment (enen92) [reply plugin.video.srf_ch_replay/addon.py]: I think there is no other way, the only option is to define a default view for your container content on the skin settings. That applies to all content including add-ons.

Comment (ManBehindMooN) [reply plugin.video.srf_ch_replay/addon.py:90]: @enen92: Is there an allowed way for that? What I wanna basically do is to have the InfoWall ViewType when I'm in the add-on. Right now the default seems to be the list and for every new view or change a page I have to set the view type again. If there is a way could you please refer me to the documentation? Right now this was the only way I have found... Thank you for your help... if there is no way... I will remove it...

Comment (ManBehindMooN) [reply plugin.video.srf_ch_replay/addon.py]: ok, then I remove it... thanks for the information

**Review by enen92 — APPROVED** [review]

**Pull request merged**